tests/system/small/ml/test_llm.py

```diff
@@ -383,3 +384,11 @@ def test_llm_gemini_pro_score_params(llm_fine_tune_df_default_index):
         ],
         index=6,
     )
+
+
+def test_palm2_text_embedding_deprecated():
+    with pytest.warns(exceptions.ApiDeprecationWarning):
+        try:
+            llm.PaLM2TextEmbeddingGenerator()
+        except (Exception):
```

Comment:
It feels like we are also ignoring all the exceptions? I wonder if we could just let it be raised and fail this test, if such a case occurs.

Reply:
The backend is already unavailable now. The exception is expected. We only want to check the warnings.